build/configuration/pom.xml

```diff
@@ -215,13 +215,13 @@
 
       <version.smallrye-mutiny>1.3.1</version.smallrye-mutiny>
       <!-- Defaults to Spring 5 and Spring Boot 2 -->
-      <version.spring>5.3.24</version.spring>
-      <version.spring.boot>2.7.7</version.spring.boot>
-      <version.spring.session>2.7.0</version.spring.session>
+      <version.spring>5.3.25</version.spring>
+      <version.spring.boot>2.7.9</version.spring.boot>
+      <version.spring.session>2.7.1</version.spring.session>
```

Comment:
There is no spring session 2.7.1 in Maven Central https://repo1.maven.org/maven2/org/springframework/session/spring-session-core/

Reply:
oh I saw it in github as a latest release

https://github.com/spring-projects/spring-session/tree/2.7.1

I will revoque the change